docs/guides/Fulfillment/setting-up-white-label-seller-as-pickup-point.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Setting up white label seller as pickup point"
+slug: "setting-up-white-label-seller-as-pickup-point"
+hidden: false
+createdAt: "2023-07-25T20:30:14.043Z"
+updatedAt: "2023-07-25T20:30:14.127Z"
+---
+
+Every [franchise account](https://help.vtex.com/en/tutorial/what-is-a-franchise-account--kWQC6RkFSCUFGgY5gSjdl) created in VTEX is automatically associated as a [white label seller](https://help.vtex.com/en/tutorial/white-label-seller--5orlGHyDHGAYciQ64oEgKa) of the main account. So [pickup points](https://help.vtex.com/en/tutorial/pickup-points--2fljn6wLjn8M4lJHA6HP3R) configured in the franchise account will be valid for orders placed in the main account. The franchise account will act as white label seller of the main account.
+
+>❗ All settings must be made in the franchise account, which is the white label seller. Items will only be displayed at Checkout for shoppers with the pickup point option if the SKU is available in the inventory of the main account and also in the inventory of the franchise account.
```

Comment:
1. All settings = todas todas mesmo? 
All settings de pickup point? Tem link pra explicar o que é configurado?

2. _Items_ do catálogo do white label seller/franchise account?

3. _Checkout_ da main account?

4. A construção gramatical de "shoppers **with** the pickup option" dá a entender que a posse/relação de pickup options é do shopper e não do items no checkout. 

5. Como faz pra associar o SKU na conta franquia e main account? Por ser seller white label, como é feito o envio? É necessário fazer o processo de match received SKUs? Que endpoints tão envolvidos nesse processo?

6. Me parece que tem tanta info importante pra esse processo acontecer que não foi informada nesse callout, que vale um heading inteiro só pra isso

Reply:
Enderecei vários desses pontos! Como era uma doc do Help Center, carece de info sobre endpoints. Acrescentei várias, mas ainda precisaria de mais, isso só não está priorizado dentro dessa tarefa e da relevância desse material no momento.